xwiki-platform-core/xwiki-platform-help/xwiki-platform-help-ui/src/main/resources/XWiki/XWikiSyntaxDefinitionLists.ru.xml

```diff
@@ -0,0 +1,115 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Looks like a duplicate of the en version. Is this expected ?

Reply:
AFAIR we use the English file to create the new translation and then remove unnecessary tags like attachments and objects. I guess we can change that too

Reply:
No I meant the content and title seems to be the same as the English version.

Reply:
`XWiki синтаксис список определений` seems quite Russian to me but indeed the content is in English. As I was saying in another comment, I used the English content for this translation to keep the translated title and avoid having an empty content. The content string is still marked for edit in Weblate so that contributors can do the translation.